src/quo2/components/wallet/account_overview/style.cljs

```diff
@@ -0,0 +1,86 @@
+(ns quo2.components.wallet.account-overview.style)
+
+(def account-overview-wrapper
+  {:width           335
+   :height          78
+   :align-items     :center
+   :justify-content :center})
+
+(defn account-name
+  [color]
+  {:margin-bottom 3
+   :color         color})
+
+(def reveal-icon
+  {:margin-left 4
+   :margin-top  2})
+
+(def current-value
+  {:margin-bottom 2})
+
+(def row-centered
+  {:flex-direction  :row
+   :justify-content :center})
+
+(defn bottom-time-text
+  [margin-right?]
+  {:margin-right (when margin-right?
+                   8)})
+
+(def right-arrow
+  {:margin-top   1.25
```

Comment:
Why 1.25? I thought we always use integers?

Reply:
To make match design as closest as I could.

Reply:
yeah it must be integer